checklist.md

```diff
@@ -0,0 +1,32 @@
+# Check Your Code Against the Following Points
+
+## Code Style
+
+Make sure that you provide message when exception rises.
+
+## Code Efficiency
+
+### Make sure that your methods work with **O(1)** time complexity.
+
+The exceptions:
+- `__init__` and `clear` methods 
+- handling the collision in any method
+- resizing the dictionary
+
+Bad example:
```

Comment:
you forgot a good one

Reply:
it is nof fair to give tham the solution, no?